src/test/emulators/storage/rules/config.spec.ts

```diff
@@ -1,39 +1,45 @@
+import * as fs from "fs";
 import * as path from "path";
 import { expect } from "chai";
-import { readFileSync } from "fs-extra";
 import { tmpdir } from "os";
-import { v4 as uuidv4 } from "uuid";
 
-import { Config } from "../../../../config";
 import { Options } from "../../../../options";
 import { RC } from "../../../../rc";
 import { getStorageRulesConfig } from "../../../../emulator/storage/rules/config";
 import { StorageRulesFiles } from "../../fixtures";
 import { Persistence } from "../../../../emulator/storage/persistence";
 import { FirebaseError } from "../../../../error";
+import { RulesConfig } from "../../../../emulator/storage";
+import { SourceFile } from "../../../../emulator/storage/rules/types";
 
 const PROJECT_ID = "test-project";
 
 describe("Storage Rules Config", () => {
-  const tmpDir = `${tmpdir()}/${uuidv4()}`;
+  const tmpDir = fs.mkdtempSync(path.join(tmpdir(), "storage-files"));
```

Comment:
I think this pattern is happening regularly enough to warrant pulling out to a helper file somewhere

Reply:
Done.